website/addons/box/serializer.py

```diff
@@ -0,0 +1,110 @@
+from website.addons.base.serializer import OAuthAddonSerializer
+
+from website.util import api_url_for, web_url_for
+
+from box.client import BoxClient, BoxClientException
+
+
+class BoxSerializer(OAuthAddonSerializer):
+
+    def credentials_owner(self, user_settings=None):
+        return user_settings.owner or self.user_settings.owner
+
+    def serialize_folder(self, folder):
```

Comment:
Looks like this method is no longer being used -- can you delete this along with `test_utils/test_serialize_folder'?